upi/vsphere/ipam/cidr_to_ip.sh

```diff
@@ -33,22 +32,24 @@ is_ip_address() {
 
 get_reservation() {
   reservation=$(curl -s "http://${ipam}/api/getIPs.php?apiapp=address&apitoken=${ipam_token}&domain=${hostname}")
-  if [[ "${reservation}" == "[]" ]]; then echo ""
+  if [[ "${reservation}" == "[]" ]]; then
+    echo "empty"
   else
     reserved_ip=$(echo "${reservation}" | jq -r ".\"${hostname}\"")
-    if [ "$(is_ip_address "${reserved_ip}")" == "false" ]; then echo ""
-    else echo "$reserved_ip"
+    if [ "$(is_ip_address "${reserved_ip}")" == "false" ]; then
+      echo "malformed data: ${ip_address}"
+    else
+      echo "$reserved_ip"
     fi
   fi
 }
 
 function produce_output() {
-  if [[ "${network}" == "null" ]]
-  then
+  if [[ "${network}" == "null" ]]; then
     jq -n \
       --arg ip_address "$(get_reservation)" \
       '{"ip_address":$ip_address}'
-	exit 0
+	  exit 0
```

Comment:
why the extra space? seems like it should be dedented, not indented?

Reply:
thanks, my editor had wrong number of spaced per tab configured here. fixed!